packages/app/src/cli/models/extensions/specifications/flow_action.ts

```diff
@@ -80,5 +80,3 @@ const flowActionSpecification = createExtensionSpecification({
     }
   },
 })
-
-export default flowActionSpecification
```

Comment:
Every local specification must be declare in it's own file and should be exported as `default`. CLI [loads](https://github.com/Shopify/cli/blob/eh/sync-trigger-sdl/packages/app/src/cli/models/extensions/load-specifications.ts#L45) the list of local specifications from `.ts` files inside the folder `specifications`

In case there is no `default` the array of local extension will include an empty object and this error is received
```

╭─ error ───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╮
│                                                                                                                                                                                               │
│  Cannot read properties of undefined (reading 'identifier')                                                                                                                                   │
│                                                                                                                                                                                               │
│  To investigate the issue, examine this stack trace:                                                                                                                                          │
│    at (app/src/cli/services/generate/fetch-extension-specifications.ts:66)                                                                                                                    │
│      const remoteSpec = remote.find((remote) => remote.identifier === spec.identifier)                                                                                                        │
│    at find                                                                                                                                                                                    │
│    at (app/src/cli/services/generate/fetch-extension-specifications.ts:66)                                                                                                                    │
│      const remoteSpec = remote.find((remote) => remote.identifier === spec.identifier)                                                                                                        │
│    at map                                                                                                                                                                                     │
│    at mergeLocalAndRemoteSpecs (app/src/cli/services/generate/fetch-extension-specifications.ts:65)                                                                                           │
│      const updated = local.map((spec) => {                                                                                                                                                    │
│    at fetchSpecifications (app/src/cli/services/generate/fetch-extension-specifications.ts:57)                                                                                                │
│      const updatedSpecs = mergeLocalAndRemoteSpecs(local, extensionSpecifications)                                                                                                            │
│    at async generate (app/src/cli/services/generate.ts:44)                                                                                                                                    │
│      const specifications = await fetchSpecifications({token, apiKey, config: options.commandConfig})                                                                                         │
│    at run (app/src/cli/commands/app/generate/extension.ts:98)                                                                                                                                 │
│      await generate({                                                                                                                                                                         │
│                                                                                                                                                                                               │
╰───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────╯
```